source/extensions/filters/http/composite/filter.cc

```diff
@@ -93,7 +93,8 @@ void Filter::onMatchCallback(const Matcher::Action& action) {
     ENVOY_LOG(debug, "failed to create delegated filter {}",
               accumulateToString<absl::Status>(
                   wrapper.errors_, [](const auto& status) { return status.ToString(); }));
-    return;
+
+    return Matcher::MatchCallbackStatus::StopAndFailStream;
```

Comment:
Is this a behavior change? What would occur prior  ( I think it just continued). If it will now fails and sends local reply, we should guard https://github.com/envoyproxy/envoy/pull/27906/files#diff-a3028f4528e8571bd956088a4229c38aa5257c1f5682f740a97b2d524ca3f7b2R145 with a runtime flag.

Reply:
To address this issue, I added the runtime flag `envoy.reloadable_features.enhanced_match_callback`.
If something goes wrong, the "old path" will return `Matcher::MatchCallbackStatus::Continue` so it doesn't stop the filter chain iteration and continue despite the problems.

Reply:
yep and include the flag name in the line. Thanks